noxfile.py

```diff
@@ -102,6 +119,7 @@ def autotyping(session: nox.Session) -> None:
     """
     session.run_always("pdm", "install", "-dG", "codemod", external=True)
 
+    load_env_var("LIBCST_PARSER_TYPE", default="native")
```

Comment:
why not do this at the top level?

Reply:
https://github.com/DisnakeDev/disnake/pull/969/commits/7986a840205b6fb7186b03a02cafd0aa0823a801